src/components/estimator.js

```diff
@@ -26,10 +25,10 @@ var Estimator = React.createClass({
     readOnly: React.PropTypes.bool,
     itemType: React.PropTypes.string.isRequired,
     score: React.PropTypes.string.isRequired,
-    estimateChanger: React.PropTypes.object.isRequired
+    estimateChanger: React.PropTypes.object
```

Comment:
As this is now not required, we might get an error on line 77 where we don't check to see if it exists before we call things on it?